workers/unity/Packages/com.improbable.gdk.core/CoreInitialization.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core
+{
+    public static class CoreInitialization
```

Comment:
I'd call this something else. Maybe `WorldsInitializationHelper` or something as it doesn't initialise the core.